checklists/apim_checklist.en.json

```diff
@@ -149,6 +150,7 @@
             "waf": "Reliability",
             "guid": "9c8d1664-dd9a-49d4-bd83-950af0af4044",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = ( sku.name == "Premium" and  isnotnull(zones) and sku.capacity >= 2 ) | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes